src/utils/pages.ts

```diff
@@ -15,11 +15,16 @@ limitations under the License.
 */
 
 import { ConfigOptions } from "../SdkConfig";
+import { getEmbeddedPagesWellKnown } from '../utils/WellKnownUtils';
 
 export function getHomePageUrl(appConfig: ConfigOptions): string | null {
     const pagesConfig = appConfig.embeddedPages;
     let pageUrl = pagesConfig?.homeUrl;
 
+    if (!pageUrl) {
+        pageUrl = getEmbeddedPagesWellKnown()?.home_url;
```

Comment:
this should be after the config fallback

Reply:
@kittykat explicitly requested in https://github.com/matrix-org/matrix-react-sdk/pull/7790#pullrequestreview-968105088 to prefer config.json and only to use .well-known if config.json does not have a homeUrl.

Reply:
the statement below is more config.json though?

Reply:
You are correct. It is now fixed.